modules/nlu/src/views/full/entities/ListEntity.tsx

```diff
@@ -132,15 +144,28 @@ export const ListEntityEditor: React.FC<Props> = props => {
           />
         </FormGroup>
         {state.occurrences.length > 0 && (
-          <FormGroup label={lang.tr('module.nlu.entities.occurrenceLabel')}>
-            {state.occurrences.map((o, i) => (
-              <Occurrence
-                key={o.name}
-                occurrence={o}
-                remove={removeOccurrence.bind(null, i)}
-                onChange={editOccurrence.bind(null, i)}
-              />
-            ))}
+          <FormGroup
+            label={
+              <span>
+                <Tooltip position={Position.LEFT} popoverClassName={style.configPopover}>
+                  <span>
+                    {lang.tr('nlu.entities.occurrenceLabel')}&nbsp;
+                    <Button icon="sort" minimal text="" onClick={sortOccurernces} />
```

Comment:
why sort ? do you have an endless list of occurrences for a given entity so it's hard to use ?

Reply:
Sorting was mentioned as an important feature in https://github.com/botpress/botpress/issues/2831
Should I remove it?